tests/quasi_hyperbolic_momentum_sgd_test.cpp

```diff
@@ -0,0 +1,53 @@
+/**
+ * @file quasi_hyperbolic_momentum_sgd_test.cpp
+ * @author Niteya Shah
+ *
+ * ensmallen is free software; you may redistribute it and/or modify it under
+ * the terms of the 3-clause BSD license.  You should have received a copy of
+ * the 3-clause BSD license along with ensmallen.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#include <ensmallen.hpp>
+#include "catch.hpp"
+
+using namespace ens;
+using namespace ens::test;
+/*
+* Tests the Quasi Hyperbolic Momentum SGD update policy.
+*/
+TEST_CASE("QHSGDSpeedUpTestFunction", "[QHMomentumSGDTest]")
```

Comment:
The name `SpeedUp` probably isn't meaningful here, just `QHSGDTestFunction` should be fine.